packages/bpk-component-pagination/src/bpk-pagination-page.scss

```diff
@@ -23,3 +23,35 @@
   height: $bpk-spacing-xl;
   padding: 0;
 }
+
+.bpk-pagination-page--selected {
+  cursor: auto;
```

Comment:
this code needs room to breath 😅  - stylelint will complain soon I suspect.

Reply:
I've been betrayed by VSCode, the inline linting did not complain so I thought I could get away with it </3